### PR TITLE
Real time pricing and add/edit note features

### PR DESCRIPTION
**Pull request by Jayendra88**

### CODE REVIEW

**Comment by Jayendra88** [comment]
> Hi! I'm VTEX IO CI/CD Bot and I'll be helping you to publish your app! 🤖
> 
> Please select which version do you want to release:
> 
> * [ ]  Patch (backwards-compatible bug fixes)
> * [x]  Minor (backwards-compatible functionality)
> * [ ]  Major (incompatible API changes)
> 
> And then you just need to merge your PR when you are ready! There is **no need** to create a release commit/tag.
> 
> * [ ]  No thanks, I would rather do it manually 😞

File: manifest.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "vendor": "whitebird",
+  "vendor": "alfonsob2b",
```
[the remark this answers is not in the document]
Comment (HarshaClouda) [reply manifest.json:2]: Changed to whitebird.

File: node/clients/masterdata.ts [hunk 2]
```diff
@@ -11,73 +11,53 @@ export default class MasterDataClient extends JanusClient {
...
   public async getWishlist(id: string): Promise<Wishlist> {
-    try {
-      return await this.http.get(
-        `/api/dataentities/${DATA_ENTITY_NAME}/documents/${id}?_fields=${FIELDS}&_schema=${SCHEMA}`
-      )
-    } catch (error) {
-      return error
-    }
+    return this.http.get(
+      `/api/dataentities/${DATA_ENTITY_NAME}/documents/${id}?_fields=${FIELDS}&_schema=${SCHEMA}`
```
[the remark this answers is not in the document]
Comment (HarshaClouda) [reply node/clients/masterdata.ts:23]: Thank You.

File: node/resolvers/getWishlistsByEmail.ts [hunk 3]
```diff
@@ -1,32 +1,28 @@
...
 
-    const wishlists = await md.searchWistlist('email', email, pagination)
+  const wishlists = await md.searchWistlist("email", email, pagination);
 
-    return wishlists || []
-  } catch (error) {
-    return error
-  }
-}
+  return wishlists || [];
```
[the remark this answers is not in the document]
Comment (HarshaClouda) [reply node/resolvers/getWishlistsByEmail.ts:27]: Fixed

File: react/components/Notes.jsx [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+
+  const wishlistRef = useRef(initialWishlist);
+
+  const { showToast } = useContext(ToastContext);
+
+  useEffect(() => {
+    if (currentNotes) {
+      setNotes(currentNotes);
+    }
+  }, [currentNotes]);
```
[the remark this answers is not in the document]
Comment (HarshaClouda) [reply react/components/Notes.jsx:28]: Removed

File: react/components/Notes.jsx [hunk 5]
```diff
@@ -0,0 +1,182 @@
...
+  skuReference,
+  currentNotes,
+  productName,
+  productImage,
+  partNumber,
+  price,
+  currency,
+}) => {
+  const [show, setShow] = useState(false);
+  const [notes, setNotes] = useState("");
```
[the remark this answers is not in the document]
Comment (HarshaClouda) [reply react/components/Notes.jsx:18]: Fixed

File: react/components/Notes.jsx [hunk 6]
```diff
@@ -0,0 +1,182 @@
...
+    wishlistRef.current = initialWishlist;
+  }, [initialWishlist]);
+
+  const handleNotesSubmit = async () => {
+    try {
+      const updatedProducts = wishlistRef.current.products.map((product) => {
+        if (product.skuCodeReference === skuReference) {
+          product.notes = notes;
+
+          return product;
```
[the remark this answers is not in the document]
Comment (HarshaClouda) [reply react/components/Notes.jsx:40]: Removed

File: react/components/Notes.jsx [hunk 7]
```diff
@@ -0,0 +1,182 @@
...
+            strokeLinejoin="round"
+          />
+          <path
+            d="M8 17H16"
+            stroke="#292D32"
+            strokeWidth="1.5"
+            strokeLinecap="round"
+            strokeLinejoin="round"
+          />
+        </svg>
```
[the remark this answers is not in the document]
Comment (HarshaClouda) [reply react/components/Notes.jsx:107]: Done

File: react/components/SearchSKU/QuickOrderAutocomplete.tsx [hunk 8]
```diff
@@ -161,7 +163,18 @@ const QuickOrderAutocomplete: FunctionComponent<
...
...tRAAAgEDAwIEAwUFBAQAAAF9AQIDAAQRBRIhMUEGE1FhByJxFDKBkaEII0KxwRVS0fAkM2JyggkKFhcYGRolJicoKSo0NTY3ODk6Q0RFRkdISUpTVFVWV1hZWmNkZWZnaGlqc3R1dnd4eXqDhIWGh4iJipKTlJWWl5iZmqKjpKWmp6ipqrKztLW2t7i5usLDxMXGx8jJytLT1NXW19jZ2uHi4+Tl5ufo6erx8vP09fb3+Pn6/8QAHwEAAwEBAQEBAQEBAQAAAAAAAAECAwQFBgcICQoL/8QAtREAAgECBAQDBAcFBAQAAQJ3AAECAxEEBSExBhJBUQdhcRMiMoEIFEKRobHBCSMzUvAVYnLRChYkNOEl8RcYGRomJygpKjU2Nzg5OkNERUZHSElKU1RVVldYWVpjZGVmZ2hpanN0dXZ3eHl6goOEhYaHiImKkpOUlZaXmJmaoqOkpaanqKmqsrO0tba3uLm6wsPExcbHyMnK0tPU1dbX2Nna4uPk5ebn6Onq8vP09fb3+Pn6/9oADAMBAAIRAxEAPwD9Sv7Pi/vT/wDf+T/4qj+z4v70/wD3/k/+KrD+I3xF0D4T+CtU8V+J79dN0TTY/MnnYFjyQqqqjlmZiFAHUkV4H/w118QF0f8A4So/s8+LB4C2+f8A2n/aFt/aHkdfN/s//WY2/N97Hv3rpp4erWXNBaeqXy13ZlKpCDsz6Y/s+L+9P/3/AJP/AIqj7BF/fm/7/v8A41i/Dz4g6D8VPBeleKvDN+upaJqUXm29woIPUhlYHlWVgVKnkEEV0dYSi4txlo0aJpq6Pmn9vixmX4T+FtfkspdS0Hwx4v0vXNds4kMhm0+J2EwKfxKC6kjoApJ4Fe0P8XPBUfgL/hNm8U6WPCfk+eNX+1L5BXbnAbP3sfw/ezxjPFdXNDHcwyQzRrLFIpR43UFWUjBBB6givDG/YZ+A7+IjrZ+Gmkm9LlzHul+zZJz/AMe+/wArHtsxXZCpSnTjTq3XK3tZ7+rX36+hhKM4ycoW17nIf8E79Tsdd+Efi/V9JlhXQ9U8aatfaZYRupaxtXdPLhdAf3Z4LbT2cHoRX1LXz1+zx+xroP7NvxI8ZeI/DWv6k2i69Gsdv4dk/wCPezAbcSW3EykHIRiAVVmBLE5r6FpY2dOpXlOk7p/1b5DoRlGmozVmgoooriNwooooA//Z" />
```
[the remark this answers is not in the document]
Comment (HarshaClouda) [reply react/components/SearchSKU/QuickOrderAutocomplete.tsx:174]: Done

File: react/components/Wishlist.jsx [hunk 9]
```diff
@@ -283,86 +286,183 @@ function Wishlist({ wishlists, fetchData }) {
...
-                      event,
-                      createWishlist,
-                      userEmail,
-                      setFieldValidationTable,
-                      nameListAccountTable,
-                      setNameListAccountTable,
-                      setIsModalAccountTable,
-                      fetchData
-                    )
+          {deviceInfo.type === 'phone' ?
```
[the remark this answers is not in the document]
Comment (HarshaClouda) [reply react/components/Wishlist.jsx:289]: Done

**Pull request merged**